pkg/subscription/subscription_test.go

```diff
@@ -0,0 +1,196 @@
+package subscription
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/osbuild/images/internal/common"
+	"github.com/osbuild/images/pkg/blueprint"
+	"github.com/stretchr/testify/assert"
+)
+
+// rhsmConfigNotSamePointers is a helper function to check that the pointers in the
+// two RHSMConfig objects are not the same.
+func rhsmConfigNotSamePointers(t *testing.T, c1, c2 *RHSMConfig) {
```

Comment:
Nitpick: this is unused here. Maybe it could be moved to the commit that adds the cloning and tests for this?

Reply:
Thanks, I probably missed the function while splitting the commit. :+1:

Reply:
Should be fixed now.